docs/how_to_contribute.md

```diff
@@ -15,85 +15,117 @@ And there are a lot of domains.
+Before contributing to Fundus you should check if you have Fundus installed in `editable` mode and using dev-requirements.
 If you haven't done this so far or aren't sure about you should
 
 1. Clone the repository
-2. Optional but recommended: Create a virtualenv or conda environment
+2. Optional but recommended: Create a virtual environment (.venv) or conda environment
 3. Navigate to the root of the repository
 4. Run pip install -e .[dev]
 
 ## How to add a Publisher
 
-Before contributing a parser, check the [**readme**](../README.md) if there is already support for your desired publisher.
+Before contributing a parser, check the [**supported publishers**](supported_publishers.md) table if there is already support for your desired publisher.
 In the following, we will walk you through an example implementation of the [*Los Angeles Times*](https://www.latimes.com/) covering the best practices for adding a news source.
 
 ### 1. Creating a Parser Stub
 
 Take a look at the file structure in `fundus/publishers`.
-Fundus is divided into country-specific sections representing the country a news source originates from.
-For example
+Publishers covered are divided into country-specific sections representing the country a news source originates from.
+As abbreviations for countries, we use the [**ALPHA-2**](https://www.iban.com/country-codes) codes described in ISO 3166.
+For example:
 
-- `fundus/publishers/de/` for German publishers and
-- `fundus/publishers/us/` for American publishers
+- `fundus/publishers/de/` for German publishers
+- `fundus/publishers/us/` for US publishers
 - ...
 
-For the Los Angeles Times, the correct location is `fundus/publishers/us/los_angeles_times.py` since they are an American publisher.
-In the newly created file, add an empty parser class inheriting from `BaseParser`.
+Now create an empty file in the corresponding country section using the publishers' name - or some kind of abbreviation - as the file name.
+For the Los Angeles Times, the correct country section is `fundus/publishers/us/`, since they are a newspaper based in the United States, with a filename like `la_times.py` or `los_angeles_times.py`.
+We will continue here with `la_times.py`.
+In the newly created file, add an empty parser class inheriting from `ParserProxy` and a parser version `V1` subclassing `BaseParser`.
 
 ``` python
-from fundus.parser import BaseParser
+from fundus.parser import ParserProxy, BaseParser
 
-class LosAngelesTimesParser(BaseParser):
-    pass
+
+class LATimesParser(ParserProxy):
+    class V1(BaseParser):
+        pass
 ```
 
+Internally, the `ParserProxy` maps crawl dates to specific versions (`V1`, `V2`, etc.) subclassing `BaseParser`.
+Since Fundus' parsers are written by hand and in most cases bound to the layout they were written for, Fundus takes the extra proxying step to address changes to the layout.
+Don't worry about it now.
+To add a new parser you don't even have to know why Fundus does things this way.
```

Comment:
I am not sure if this belongs here. Some might consider it patronizing. In my opinion, we should provide a surface level explanation and link to in-depth one.

Reply:
Agree